### DISCUSSION TITLE
Improve Product Page - Category Section

### DISCUSSION THREAD

**Discussion by krrisharora** [inspireui/support#5795] [New Feature/Ideas Requests]
On the current product page, there is a section that show product category like this:

<img width="282" alt="image" src="https://github.com/user-attachments/assets/273375dd-b2f6-4d4e-a4e8-4e2bc7bbf12c">

I want that it should display a more vibrant look, something like this:

<img width="583" alt="image" src="https://github.com/user-attachments/assets/cbb0b29d-e65c-4005-a9bd-a039aede89c0">

This will make the product page more attractive and understantable , rather than the text.

**Comment by osman-kocak** [comment]
Its best design for categorys
